proofs/pfedit.ml

```diff
@@ -233,10 +233,10 @@ let declare_implicit_tactic tac = implicit_tactic := Some tac
 
 let clear_implicit_tactic () = implicit_tactic := None
 
-let solve_by_implicit_tactic env sigma evk =
+let apply_implicit_tactic tac = (); fun env sigma evk ->
```

Comment:
No, even as of 4.04 OCaml is still doing badly with closures. I think that Flambda does mitigate the issue though. Here, it is probably a case of premature optimization of mine, as I tend to be explicit about the expected closure creation when it is trivial. If this startles you, I can remove this line.